### PR TITLE
[Kijiji.ca] New ruleset

### PR DESCRIPTION
**Pull request by youdly**

### CODE REVIEW

File: src/chrome/content/rules/Kijiji.ca.xml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+		- kijijity	(s)
+		- kijinaute	(s)
+		- m	(t)
+		- manage	(t)
+		- mingleprivate	(t)
+		- mobile-api	(r)
+		- newfoundland	(t)
+		- rtsriak	(t)
+		- secondhandeconomy	(t)
+		- send-api	(t)
```
Comment (J0WI) [inline src/chrome/content/rules/Kijiji.ca.xml]: `tech.kijiji.ca`

File: src/chrome/content/rules/Kijiji.ca.xml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+	<target host="help.kijiji.ca" />
+	<target host="info.kijiji.ca" />
+	<target host="kit.kijiji.ca" />
+	<target host="re.kit.kijiji.ca" />
+	<target host="marketing.kijiji.ca" />
+	<target host="mingle.kijiji.ca" />
+	<target host="pint.kijiji.ca" />
+	<target host="tmx.kijiji.ca" />
+	<target host="www.kijijiblog.ca" />
+	<target host="fr.kijijiblog.ca" />
```
Comment (J0WI) [inline src/chrome/content/rules/Kijiji.ca.xml]: They're already in `Kijijiblog.ca.xml`

**Comment by youdly** [comment]
Updated comment

File: src/chrome/content/rules/Kijiji.ca.xml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+	s: self-signed certificate
+	t: timeout on https
+-->
+<ruleset name="Kijiji.ca">
+
+	<target host="kijiji.ca" />
+	<target host="www.kijiji.ca" />
+	<target host="admarkt.kijiji.ca" />
+	<target host="aide.kijiji.ca" />
+	<target host="autodiscover.kijiji.ca" />
```
Comment (J0WI) [inline src/chrome/content/rules/Kijiji.ca.xml]: That one seems to be broken.

**Comment by J0WI** [comment]
@youdly would be great to have this fixed for the upcoming release.

**Comment by youdly** [comment]
Removed `autodiscover` and rebased.

**Comment by youdly** [comment]
failed travis test on chrome beta

**Comment by J0WI** [comment]
[triggering Travis again]

**Comment by J0WI** [comment]
[triggering Travis again]

**Pull request merged**